ytmusicapi/mixins/search.py

```diff
@@ -203,12 +203,100 @@ def search(self,
                                      category))
 
             if filter:  # if filter is set, there are continuations
-                request_func = lambda additionalParams: self._send_request(
-                    endpoint, body, additionalParams)
-                parse_func = lambda contents: parse_search_results(contents, type, category)
+
+                def request_func(additionalParams):
+                    return self._send_request(endpoint, body, additionalParams)
+
+                def parse_func(contents):
+                    return parse_search_results(contents, type, category)
 
                 search_results.extend(
                     get_continuations(res['musicShelfRenderer'], 'musicShelfContinuation',
                                       limit - len(search_results), request_func, parse_func))
 
         return search_results
+
+    def get_search_suggestions(self, query: str, detailed_runs=False) -> Union[List, List[Dict]]:
+        """
+        Get Search Suggestions
+
+        :param query: Query string, i.e. 'faded'
+        :param detailed_runs: Whether to return detailed runs of each suggestion.
+            If True, it returns the query that the user typed and the remaining
+            suggestion along with the complete text (like many search services
+            usually bold the text typed by the user).
+            Default: False, returns the list of search suggestions in plain text.
+        :return: List of search suggestion results depending on detailed_runs param.
+            Examples:
+                1.
+                Params:
+                    query = 'fade'
+                    detailed_runs = False
+
+                Response:
```

Comment:
I think you need to format this differently so it gets formatted correctly as JSON in the built documentation. Please check the other docstrings and build the documentation (run `./make.bat html` on Windows or `make html` on Linux from the docs folder).